src/seeds.settings.cpp

```diff
@@ -102,10 +106,10 @@ void settings::reset() {
   // =====================================
 
   // community buiding points for referrer when user becomes resident
-  confwithdesc(name("refcbp1.ind"), 1, "Community buiding points for referrer when user becomes resident", high_impact);
+  confwithdesc(name("vou.cbp1.ind"), 1, "Community buiding points for voucher when user becomes resident", high_impact);
 
   // community buiding points for referrer when user becomes citizen
-  confwithdesc(name("refcbp2.ind"), 1, "Community buiding points for referrer when user becomes citizen", high_impact);
```

Comment:
Some code still refers to the old value here so IDK how this is passing unit tests.

      const name cbp_reward_citizen = "refcbp2.ind"_n; // hpp file

      name cbp_param = is_citizen ? cbp_reward_citizen : cbp_reward_resident; // cpp file

I made another comment down there, let's delete the code that hands out CBP for referrals (they get them with the vouching already). Then delete that variable which refers to a non-existent setting.

Reply:
Done. Check that the accounts unit test called "test remove" was checking if the used had CBP after the seconduser was removed, but since they were not `invited` (and thus vouched), they did not get CPB.